lib/APPTurtle/src/CustomRosTransport.cpp

```diff
@@ -77,10 +71,10 @@ bool CustomRosTransport::open(uxrCustomTransport* transport)
     }
     else
     {
-        const micro_ros_agent_locator* locator = reinterpret_cast<micro_ros_agent_locator*>(transport->args);
-        const int                      UDP_OK  = 1;
+        CustomRosTransport* tthis  = reinterpret_cast<CustomRosTransport*>(transport->args);
```

Comment:
I never saw this  tthis name convention before and first thought it is a typo.
Can't we be more explicit and call it "transport" or "thisTransport" to be closer to naming CS.

Reply:
We can fix it with next feature integration, this PR is already merged.